### DISCUSSION TITLE
../ path not get in Github Actions

### DISCUSSION THREAD

**Discussion by Nimeshkottege** [community/community#35864] [Actions]
In my build.gradle file There have importing the build.gradle of another project to execute below task.

task buildDocx(type: GradleBuild) {
	buildFile = "../${project-name}/build.gradle"
	tasks = ['jar']
}
 But when run CI on github actions '../' not getting correctly as a separate project.So its failing. 
 Have any idea about it?

**Comment by MitkoVtori** [comment]
The actions work only with build.gradle, you can't use ./build.gradle or ../build.gradle expect if you're making hyper links.
It's the same problem with images & others. try moving your path so you don't need to use ../

**Reply by Nimeshkottege** [reply]
@MitkoVtori Thanks for your reply. The scene is Some external integration tasks that need to be executed in my current project. I'm doing it without any issue importing that project build. gradle as above according to my local directory. the issue only happens on run-on actions. Both projects remain as private remote repositories in our organization. But no access with '../' as path declaration to that project which imports tests for my project. Do you have any clarification about that?

**Reply by MitkoVtori** [reply]
unfortunately, I don't know what to do in such a situation. I never had such a problem before. I understand from what you wrote that you are using two repositories(If I'm right). You can try combining the two repositories and order the files in such a way, that everything could work. Otherwise you can search information about how to do your things on google and maybe try to contact github(if it's possible). I hope this help, have a great day!

**Reply by Nimeshkottege** [reply]
Thanks, Mate. Your concern is appreciated. Have a great day!

**Comment by airtower-luna** [comment]
How are you checking out the projects? By default `actions/checkout` retrieves the current project, and the runner filesystem doesn't know anything about your other projects on GitHub.

**Reply by Nimeshkottege** [reply]
Yes, That's the issue. Do you have any solution for that?

**Reply by airtower-luna** [reply]
Use two `actions/checkout` steps to check out both repositories you need side by side: https://github.com/actions/checkout#checkout-multiple-repos-side-by-side

The example there looks pretty similar to what you described, with the current repository and an additional one for tools. Make sure to call your tools with the right working directory.

**Reply by Nimeshkottege** [reply]
Seems useful. Ill check on them.Thank you so much.
